includes/autoloader.php

```diff
@@ -1,314 +1,154 @@
 <?php
 namespace Elementor;
 
-if ( ! defined( 'ABSPATH' ) ) {
-	exit; // Exit if accessed directly.
-}
-
-/**
- * Elementor autoloader.
- *
- * Elementor autoloader handler class is responsible for loading the different
- * classes needed to run the plugin.
- *
- * @since 1.6.0
- */
-class Autoloader {
-
-	/**
-	 * Classes map.
-	 *
-	 * Maps Elementor classes to file names.
-	 *
-	 * @since 1.6.0
-	 * @access private
-	 * @static
-	 *
-	 * @var array Classes used by elementor.
-	 */
-	private static $classes_map;
-
-	/**
-	 * Classes aliases.
-	 *
-	 * Maps Elementor classes to aliases.
-	 *
-	 * @since 1.6.0
-	 * @access private
-	 * @static
-	 *
-	 * @var array Classes aliases.
-	 */
-	private static $classes_aliases;
-
-	/**
-	 * Run autoloader.
-	 *
-	 * Register a function as `__autoload()` implementation.
-	 *
-	 * @since 1.6.0
-	 * @access public
-	 * @static
-	 */
-	public static function run() {
-		spl_autoload_register( [ __CLASS__, 'autoload' ] );
-	}
+require_once __DIR__ . '/base/autoloader-base.php';
 
-	/**
-	 * Get classes aliases.
-	 *
-	 * Retrieve the classes aliases names.
-	 *
-	 * @since 1.6.0
-	 * @access public
-	 * @static
-	 *
-	 * @return array Classes aliases.
-	 */
+class Autoloader extends Autoloader_Base {
 	public static function get_classes_aliases() {
-		if ( ! self::$classes_aliases ) {
-			self::init_classes_aliases();
+		if ( ! static::$classes_aliases ) {
```

Comment:
The init of `classes_aliases` should be done in the base and stay private. The extending class should only provide a list of its own aliases

Reply:
Handled.